templates/repo/view_file.tmpl

```diff
@@ -91,6 +91,8 @@
 		<div class="file-view{{if .IsMarkup}} markup {{.MarkupType}}{{else if .IsPlainText}} plain-text{{else if .IsTextSource}} code-view{{end}}">
 			{{if .IsFileTooLarge}}
 				{{template "shared/filetoolarge" dict "RawFileLink" .RawFileLink}}
+			{{else if not .FileSize}}
```

Comment:
given how Go templates work (no types), I'll be waiting for the day users complain about seeing a file as empty even though it isn't simply because we forgot to set the file size somewhere.
Oh well, I'll keep this statement in mind.

Reply:
I have checked and it should have been set correctly.

If no, it is a bug and it's worth to be reported and fixed.

Reply:
Yeah, I simply meant for the future since Go templates can easily break without someone noticing.